scripts/is-the-bucket-ready.js

```diff
@@ -0,0 +1,43 @@
+const bucketName = process.argv[2];
+const username = process.argv[3];
+const password = process.argv[4];
+const couchbaseVersion = process.argv[5];
+
+const couchbaseVersionSplit = 5;
+// This error code is expected so ignore it
+const errorCodeToIgnore = 22;
+
+var couchbase = require('couchbase')
+var cluster = new couchbase.Cluster("http://127.0.0.1:8091");
+
+// If the couchbase version is 5 or greater then authenticate. For older versions this is unnecessary and will cause an error
+if (parseInt(couchbaseVersion.charAt(0), 10) >= couchbaseVersionSplit) {
```

Comment:
you were right. I could use the one recent sdk differently instead of using two sdks. the old one didn't require this authentication, but the new one does